packages/@aws-cdk/aws-codepipeline-actions/package.json

```diff
@@ -172,6 +174,11 @@
       "props-default-doc:@aws-cdk/aws-codepipeline-actions.CodeDeployEcsDeployActionProps.containerImageInputs"
     ]
   },
+  "pkglint": {
```

Comment:
What is happening here? This package builds fine today...

Reply:
Already removed, It seems that some other error affected this when I build CDK.